vars/neoDeploy.groovy

```diff
@@ -137,10 +137,19 @@ private deploy(script, utils, Map configuration, NeoCommandHelper neoCommandHelp
         }
     }
     catch (Exception ex) {
+
+        Exception suppressed
+
         if (dockerImage) {
             echo "Error while deploying to SAP Cloud Platform. Here are the neo.sh logs:"
-            sh "cat logs/neo/*"
+            try {
+                sh "cat logs/neo/*"
+            } catch(Exception e) {
+                echo "Unable to provide the logs."
+                suppressed = e
```

Comment:
Why and not here: ex.addSuppressed(e)

Reply:
@daniel-kurzynski : thanks for your hint. Your approach is much more straight-forward. I guess my mind was trapped in some troubleshooting mode while writing that code. I changed the code like suggested.